amaznot/resources/views/layouts/layout.blade.php

```diff
@@ -1,81 +1,248 @@
+  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
+
+  <!--ICONS-->
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta3/js/solid.min.js"></script>
+
+  <!--EXTERNAL STYLESHEET-->
+  <link rel="stylesheet" href="/css/styles.css">
+
+  <!--EXTERNAL SCRIPTS-->
+  <script src="/js/script.js"></script>
+
+  <!--SMOOTH SCROLLING CROSS-BROWSER-->
+  <script>
+    $(document).ready(function() {
+      // Add smooth scrolling to all links
+      $("a").on('click', function(event) {
+
+        // Make sure this.hash has a value before overriding default behavior
+        if (this.hash !== "") {
+          // Prevent default anchor click behavior
+          event.preventDefault();
+
+          // Store hash
+          var hash = this.hash;
+
+          // Using jQuery's animate() method to add smooth page scroll
+          // The optional number (800) specifies the number of milliseconds it takes to scroll to the specified area
+          $('html, body').animate({
+            scrollTop: $(hash).offset().top
+          }, 800, function() {
+
+            // Add hash (#) to URL when done scrolling (default click behavior)
+            window.location.hash = hash;
+          });
+        } // End if
       });
-      </script>
+    });
+  </script>
 
 </head>
+
 <body>
-    <!--MENU-->
-    <!--END MENU-->
+  <!--NAVBAR-->
+  <div class="custom-nav">
+    <input class="headerData" value="{{ $headerData['string'] }}" style="display: none;" />
+    <div class="custom-top-nav">
+      <div class="dropdown-container div-inline">
+        <div class="dropdown div-inline">
+          <button class="btn dropdown-toggle shadow-none lang" type="button" id="dropdownMenuButton" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+            EN
+          </button>
+          <div class="dropdown-menu" aria-labelledby="dropdownMenuButton">
+            <a class="dropdown-item change-dropdown" href="#">EN</a>
+            <a class="dropdown-item change-dropdown" href="#">FR</a>
+          </div>
+        </div>
+        <div class="dropdown div-inline">
+          <button class="btn dropdown-toggle shadow-none currency" type="button" id="dropdownMenuButton" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+            CAD
+          </button>
+          <div class="dropdown-menu" aria-labelledby="dropdownMenuButton">
+            <a class="dropdown-item change-dropdown" href="#">CAD</a>
+            <a class="dropdown-item change-dropdown" href="#">USD</a>
+          </div>
+        </div>
+      </div>
+      <div class="controller-container div-inline">
+        <div class="profile div-inline mr-4">
+          <a class="controller-link" href="{{ route('login') }}">
```

Comment:
The login button here for the desktop version of the site does not have auth checks, and the desktop header is also missing buttons like logout, cart, and admin page